Makefile

```diff
@@ -69,6 +73,15 @@ $(TESTS) $(TOOLS) $(EXAMPLES): $(CCTZ_LIB)
 
 $(TESTS:=.o) $(TOOLS:=.o) $(EXAMPLES:=.o):
 
+$(CCTZ_SHARED_LIB): $(CCTZ_OBJS)
+	$(CC) $(LDFLAGS) -shared -Wl,-soname,$(CCTZ_SHARED_SONAME) -o$@ $(CCTZ_OBJS)
+
+$(CCTZ_SHARED_SONAME): $(CCTZ_SHARED_LIB)
```

Comment:
Is it important for these links to be in the build tree, or are they best handled by the installation rule?